src/plugins/index_pattern_editor/public/components/form_schema.ts

```diff
@@ -18,7 +18,7 @@ export const schema = {
     defaultValue: '',
     helpText: i18n.translate('indexPatternEditor.validations.titleHelpText', {
       defaultMessage:
-        'Use an asterisk (*) to match multiple characters. Spaces and the characters , /, ?, ", <, >, | are not allowed.',
+        'Enter an index pattern string that matches one or more data sources. Use an asterisk (*) to match multiple characters. Spaces and the characters , /, ?, ", <, >, | are not allowed.',
```

Comment:
Is "index pattern" needed, or can you simply say "string".  Also suggest shortening the second sentence a bit.

Enter a string that matches one or more data sources. An asterisk (*) matches multiple characters. Spaces and the characters , /, ?, ", <, >, | are not allowed.',

Reply:
>Is "index pattern" needed

Index pattern is a specific technical term used elsewhere in kibana and elasticsearch and I think its important to employ it here.

Reply:
or could the label for the field be "Index pattern name"

Reply:
@gchaps What if its labeled 'Name (index pattern)'? 

Its unfortunate that the name of the data view is also an index pattern. Its awkward to explain.

Reply:
You're right.  It's better to put "index pattern" in the description. "index pattern string" sounds like such a mouthful.  

Can "string" be removed because by definition it's a string?

> Enter an index pattern that matches one or more data sources.